content/en/docs/concepts/configuration/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Poseidon-Firmament - An alternate scheduler
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+ 
+Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) integration for Kubernetes. At a very high level, Poseidon/Firmament scheduler augments the current Kubernetes scheduling capabilities by incorporating a new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. It models the scheduling problem as a constraint-based optimization over a flow network graph – by reducing scheduling to a min-cost max-flow optimization problem.
+
```

Comment:
* I'd rather read one or two sentences, at most, in the overview section.
* Can you break up or clean up this sentence:
At a very high level, the Poseidon scheduler augments the current Kubernetes scheduling capabilities by incorporating a new novel flow network [graph-based scheduling capabilities alongside the default Kubernetes Scheduler?????]

Reply:
We have simplified the language and broken down the long sentences to shorter ones.
Please check.

Reply:
* I just read the design doc. Is Poseidon really in the alpha state in 1.13 or will it be in 1.14?
* I'm confused: Is Poseidon encapsulating Firmament with the integration? What version of Poseidon includes what version of Firmament? Or the scheduler and integration are separate installs?
* nit, suggested change: Poseidon is the integration of the Firmament scheduler with Kubernetes. 
* nit: Unless I have missed something, the overview and the introduction repeat the same information, verbatim.
* From the design doc, a good explanation of firmament and the role of Poseidon: Firmament models workloads and clusters as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
* nit: use of new and novel is redundant?
It incorporates new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. 
Possible change: The Firmament scheduler [or Poseidon integration?] incorporates novel flow network, graph-based scheduling capabilities ...
Do you need the graph-based?

Reply:
1. Poseidon-Firmament is supported  from Kubernetes release 1.6 and works with all subsequent K8S releases. 
2. This is updated in the introduction section. _Poseidon is the integration glue for the Firmament scheduler with Kubernetes._ 
 The release process for Poseidon-Firmament scheduler is explained in detail at this [link](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/releases/release-process.md). In summary, Poseidon and Firmament are in lock-step. Everytime there is a change in Firmament or Poseidon there is a corresponding release created. for ex. The current poseidon release is 0.6 and the current corresponding firmament release is 0.6. This information has been updated in the Current Project Stage section in the document.
4. The overview is now changed.
5. Added this and reworded the introduction section. 
6. Removed new.
7.  Graph based is definitely necessary.